package.json

```diff
@@ -15,22 +15,15 @@
     "test": "yarn run test:unit",
     "lint": "eslint ."
   },
-  "engines": {
-    "node": ">=14.0.0"
-  },
   "devDependencies": {
     "@babel/core": "^7.18.5",
     "@babel/eslint-parser": "^7.18.2",
```

Comment:
`@babel/core` and `@babel/eslint-parser` are already listed in `@folio/eslint-config-stripes`. Can we remove them too?